### PR TITLE
std/time: Give an example to get UNIX_EPOCH in seconds

### PR DESCRIPTION
**Pull request by kallisti5**

### CODE REVIEW

**Comment by kallisti5** [comment]
rebased from PR #44301

File: src/libstd/time/mod.rs [hunk 1]
```diff
@@ -382,6 +382,17 @@ impl fmt::Debug for SystemTime {
...
+///
+/// # Examples
+///
+/// ```no_run
+/// use std::time::{SystemTime, UNIX_EPOCH};
+///
+/// match SystemTime::now().duration_since(UNIX_EPOCH) {
+///     Ok(n) => println!("1970-01-01 00:00:00 UTC was {} seconds ago!", n.as_secs()),
+///     Err(_) => panic!("SystemTime before UNIX EPOCH!"),
+/// };
```
Comment (GuillaumeGomez) [inline src/libstd/time/mod.rs]: Useless semicolon.

**Comment by GuillaumeGomez** [comment]
And just `push -f` on your branch, no need to recreate a PR. ;)

**Comment by kallisti5** [comment]
Oh, I didn't realize you could force a push updating a remote branch without breaking *everything* at github.. neat!   Done.

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup

**Pull request merged**